parser/fsimporter.go

```diff
@@ -218,6 +218,10 @@ func parseFiles(indexedFiles []*fpt.IndexedFile, parsingThreads int, datastore D
 	parsingWG.Wait()
 }
 
+//removeOldFilesFromIndex checks all indexedFiles passed in to ensure
+//that they have not previously been imported into the same database.
+//The files are compared based on their hashes (md5 of first 15000 bytes)
+//and the database they are slated to be imported into.
 func removeOldFilesFromIndex(indexedFiles []*fpt.IndexedFile,
```

Comment:
The name of this function is a little ambiguous since it's not clear what makes files "Old" and "Index" is also used in Mongo terminology but that is not what is being referred to here. The comment hopefully helps clear this up.